File: src/2-Problem12.cpp
#include <iostream>
#include <cstring>

using namespace std;
int main(){
	int n,hasil;

	cout<<"Masukan jumlah pemotongan : ";
	cin>> n;
	
	hasil=1;
	for (int i=1; i<=n;i++){
		hasil= hasil + i;
	}
	
	cout<<"Potongan maksimal yang didapat adalah "<< hasil<< endl;
	return 0;
}

File: src/2-Problem12.py
"""
    NIM/Nama  : 13516002/Antonio Setya
    Topik     : Pengulangan
    Deskripsi : Diberikan jumlah pemotongan sebuah pizza, mengeluarkan potongan maksimal pizza yang didapat
"""

# Input
n = int(input("Masukan jumlah pemotongan :")) # Menyimpan berapa kali pizza dipotong

# Memroses jumlah potongan. Rumus : f(n) = n + f(n-1); f(0) = 1
out = 0
for i in range (n,0,-1):
    out += i
out += 1

# Output
print("Potongan maksimal yang didapat adalah " + str(out))
